src/run_assemble.py

```diff
@@ -25,7 +25,11 @@ def main():
     build_manifest = BuildManifest.from_file(args.manifest)
     build = build_manifest.build
     artifacts_dir = os.path.dirname(os.path.realpath(args.manifest.name))
-    output_dir = os.path.join(os.getcwd(), "dist")
+    output_dir = os.path.join(
+        os.getcwd(),
+        "dist",
+        "opensearch" if build.name == "OpenSearch" else "opensearch-dashboards"
```

Comment:
Derive this from the manifest `build.name`,  add a method that does `name.lower().replace(" ", "-")` and use that in other places we do the same thing.